src/lib/update-alternatives/model/alternative.rb

```diff
@@ -58,17 +58,17 @@ def self.load(name)
       alternative = parse_to_map(raw_data.slice(0..raw_data.find_index("\n")))
       choices = raw_data.slice(raw_data.find_index("\n") + 1..raw_data.length)
       new(
-        alternative["Name:"],
-        alternative["Status:"],
-        alternative["Value:"],
+        alternative["Name"],
+        alternative["Status"],
+        alternative["Value"],
         load_choices_from(choices)
       )
     end
 
     def self.parse_to_map(alternative_data)
       alternative = {}
       alternative_data.each do |line|
-        alternative[line.split.first] = line.split.last
+        alternative[line.split(/\W/).first] = line.split.last
```

Comment:
Well it is used to avoid ":" in the key of the map used to create the alternative.
```
new(
        alternative["Name:"],
        alternative["Status:"],
        alternative["Value:"],
        load_choices_from(choices)
      )
```
I don’t know what is the best solution, use differents split or have ":" on the map's keys